.github/workflows/create-release-tag.yaml

```diff
@@ -57,12 +57,13 @@ jobs:
           fi
           echo "Creating a new tag $tag_name..."
           SHA=${{ github.event.pull_request.merge_commit_sha }}
-          response=$(curl -s -X POST -H "Authorization: token ${{ secrets.GORELEASER_GITHUB_TOKEN }}" \
+          response=$(curl -s -o response.json -w "%{http_code}" -X POST -H "Authorization: token" \
```

Comment:
The token here in `Authorization: token` is weird - that's static and won't get replaced. Is this intentional?

Reply:
Ah thanks, was testing the output when the request failed, forgot to bring this back 🙏